watch.go

```diff
@@ -90,18 +92,67 @@ func liveEvents(lr Reloader, ch chan []string) {
 	}
 }
 
+func shouldInclude(file string, excludePatterns []string) bool {
+	for _, pattern := range excludePatterns {
+		match, err := filepath.Match(pattern, file)
+
+		if err != nil {
+			fmt.Println("Bad pattern: ", pattern)
```

Comment:
A few things here. First, I do think we should pass the logger down to here, and use a Warn if there's a error from Match. Second, I wonder if we can make this more controllable by doing a Match against an empty string for each pattern when we parse the command-line args. That should give us an error if there's a bogus pattern, and we can terminate immediately. We'll still have to handle the error here just in case, but by my reading of the code it should essentially never be triggered.